solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -14,35 +13,44 @@ import {
   PermissionsResponse,
   PermissionsResponseParser
 } from '@aws/swb-app';
-import { AuditService } from '@aws/workbench-core-audit';
+import { ProjectAccessRequest } from '@aws/swb-app/lib/dataSets/projectAccessRequestParser';
+import {
+  Action,
+  AuthenticatedUser,
+  CreateIdentityPermissionsRequestParser,
+  DynamicAuthorizationService,
+  IdentityPermission
+} from '@aws/workbench-core-authorization';
+import { IdentityPermissionParser } from '@aws/workbench-core-authorization/lib/dynamicAuthorization/dynamicAuthorizationInputs/identityPermission';
```

Comment:
can you update this import now that its exported in the authZ package index file?